Tell msgfmt to use the same byte order as pocompile

Index: translate-toolkit-2.5.0/translate/storage/test_mo.py
===================================================================
--- translate-toolkit-2.5.0.orig/translate/storage/test_mo.py
+++ translate-toolkit-2.5.0/translate/storage/test_mo.py
@@ -147,7 +147,7 @@ class TestMOFile(test_base.TestTranslati
             with open(PO_FILE, 'wb') as out_file:
                 out_file.write(posource)
 
-            subprocess.call(['msgfmt', PO_FILE, '-o', MO_MSGFMT])
+            subprocess.call(['msgfmt', PO_FILE, '-o', MO_MSGFMT, '--endianness', sys.byteorder])
             subprocess.call(['pocompile', '--errorlevel=traceback', PO_FILE, MO_POCOMPILE])
 
             store = factory.getobject(BytesIO(posource))
